mo_net/samples/word2vec/__main__.py

```
@@ -0,0 +1,755 @@
+            )
+
+    @classmethod
+    def load(
+        cls,
+        source: IO[bytes] | Path,
+        training: bool = False,
+        freeze_parameters: bool = False,
+    ) -> CBOWModel:
+        if isinstance(source, Path):
+            with open(source, "rb") as f:
+                serialized = pickle.load(f)
```

Ruff (0.12.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)